File: 80.java
class Solution4 {
    public static int removeDuplicates(int[] nums) {
        int unique_index = 2;

        for (int org_index = 0 ; org_index < nums.length - 2 ; org_index++){
            if (nums[org_index] != nums[unique_index]){
                nums[unique_index] = nums[org_index];
                unique_index++;
            }
        }
        return unique_index;
    }


}

File: 80.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        unique_idx = 2
        if len(nums) <= 2:
            return 2
        for org_idx in range(2, len(nums)):
            if nums[unique_idx - 2] != nums[org_idx]:
                nums[unique_idx] = nums[org_idx]
                unique_idx += 1
        return unique_idx


s1 = Solution()
s1.removeDuplicates([0,0,1,1,1,1,2,3,3])